ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/network_services_key.schema.yml

```diff
@@ -0,0 +1,1000 @@
+                type: dict
+                description: |
+                  Enable VTEP Network diagnostics (Optional).
+                  This will create a loopback with virtual source-nat enable to perform diagnostics from the switch
+                keys:
+                  loopback:
+                    type: int
+                    convert_types:
+                      - str
+                    description: |
+                      Loopback interface number, required (when vtep_diagnotics defined)
+                    min: 2
+                    max: 2100
+                  loopback_ip_range:
+                    type: str
+                    description: |
+                      IPv4_address/Mask
+                      Loopback ip range, a unique ip is derived from this ranged and assigned
+                      to each l3 leaf based on it's unique id, ptional (loopback is not created unless loopback_ip_range or loopback_ip_pools are set)
+                  loopback_ip_pools:
+                    type: list
+                    description: |
+                      For inventories with multiple PODs a loopback range can be set per POD to avoid overlaps.
+                      This only takes effect when loopback_ip_range is not defined, ptional (loopback is not created unless loopback_ip_range or loopback_ip_pools are set)
+                    items:
+                      type: dict
+                      keys:
+                        pod:
+                          type: str
+                          description: POD name
+                        ipv4_pool:
+                          type: str
+                          description: IPv4_address/Mask
+              ospf:
+                type: dict
+                description: |
+                  Dictionary for router OSPF configuration, optional.
+                  This will create an ospf routing instance in the tenant VRF. If there is no nodes definition, the ospf instance will be
+                  created on all leafs where the vrf is deployed. This will also cause automatic ospf redistribution into bgp unless
+                  explicitly turned off with "redistribute_ospf: false".
+                keys:
+                  enabled:
+                    type: bool
+                  process_id:
+                    type: int
+                    convert_types:
+                      - str
+                    description: Default -> vrf_id
+                  router_id:
+                    type: str
+                    description: Default -> switch router_id
+                  max_lsa:
+                    type: int
+                    convert_types:
+                      - str
+                  bfd:
+                    type: bool
+                    default: false
+                  redistribute_bgp:
+                    type: dict
+                    keys:
+                      enabled:
+                        type: bool
+                        default: false
```

Comment:
default true

Reply:
Not correct. Notice the description above is about `redistribute_ospf` which is in line 204.

Reply:
so it should be true, right?